<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_pseudo_dialog</class>
 <widget class="QDialog" name="add_pseudo_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Pseudo CUIs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <item>
      <property name="text">
       <string>$ALL_EVENTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$ABORTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$CAUTION_WARNINGS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$CDH_EVENTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$COMMANDS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$ERRORS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$REDUNDANCY_EVENTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$SAFING_EVENTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$SSM_EVENTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$SYS_EVENTS</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>add_pseudo_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>add_pseudo_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
